samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -73,6 +73,8 @@
   static final String INMEMORY_KV_STORAGE_ENGINE_FACTORY =
       "org.apache.samza.storage.kv.inmemory.InMemoryKeyValueStorageEngineFactory";
 
+  public static final String CLEAN_LOGGED_STOREDIRS_ON_START = STORE_PREFIX + "container.start.clean";
```

Comment:
Is this supposed to be an internal config? If not, can you add a configuration doc for this?

Reply:
Yes this is purely internal -- intended for benchmarking and validation use.

Reply:
Would still be good to add docs (comments) explaining what this config does.

Reply:
done

Reply:
If you do plan to open it up, you can leave it as is.
Otherwise, I noticed we also insert `internal` as part of the config name to call out its internal. e.g. `samza.internal.execution.plan`